packages/manager/modules/pci/src/projects/project/kubernetes/details/service/reset-kubeconfig/reset-kubeconfig.routing.js

```diff
@@ -0,0 +1,25 @@
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider.state(
+    'pci.projects.project.kubernetes.details.service.reset-kubeconfig',
+    {
+      url: '/reset-kubeconfig',
+      params: {
+        clusterStatus: null,
+      },
+      views: {
+        modal: {
+          component: 'pciProjectKubernetesServiceResetKubeconfig',
+        },
+      },
+      layout: 'modal',
+      resolve: {
+        clusterStatus: /* @ngInject */ ($transition$) => {
```

Comment:
Isn't `clusterStatus` inherited from parent state within a `cluster` resolve or something similar ?

Reply:
The cluster object was being passed from the parent. The status is not being retrieved from the cluster object instead. Thanks